### PR TITLE
Update macOS install (wip)

### PR DESCRIPTION
**Pull request by geealbers**
This updates the install instructions for the monorepo and node module changes that are currently being made.

@Erin-Cecele I've only edited the top bit and the macOS instructions so far, but can you take a look before I go further? I'm trying out offering two install versions, a Quick and a Full. What do you think about that? 

FYI, there will also be a third option that we can offer at some point, which will hopefully be a quick way to do a full install. This is the "homebrew formula" that Matthew's been talking about. But even when it's ready, we'll initially only be able to offer it to a smaller subset of users so that we can test it out and make sure it's stable enough to release more broadly.

### CODE REVIEW

File: content/documentation/install-uninstall.md [hunk 1]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
 
-1. **Apple’s Xcode** is a set of developer tools for your mac. More information can be found [here](https://en.wikipedia.org/wiki/Xcode). Install Apple's Xcode by copying and pasting the following command and pressing enter. If Xcode is not already installed, an additional alert notification will pop up. Click “Install” and follow the prompts.
+### Quick Install
 
-    ```text
-    xcode-select --install
-    ```
+If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 2–4 in the Full Install guidelines below.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:18]: I like the concept of breaking it apart. I think this works well.

File: content/documentation/install-uninstall.md [hunk 2]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 ---
 
 {{< q-class "box warning" >}}
-- Quire is in a closed beta. Installation will not work unless you have [requested and been given access](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
+- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open source software. For a free license to use Quire for your publication projects (and for additional access to support from Getty staff) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:9]: I like this, but I want to discuss this messaging a little more and what it means when we say signing up means "they have access to Getty staff."

**Review by Erin-Cecele — COMMENTED** [review]
Overall, I like these changes a lot. A few notes and requests for clarification below! Some of the changes I was working on elsewhere in the docs are affected by my CLI question.

File: content/documentation/install-uninstall.md [hunk 3]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
 {{< /q-class >}}
 
 ## macOS Installation
 
-Open your Terminal {{< q-def "command-line shell" >}} (found in your Applications/Utilities folder or by pressing Command–Space Bar and type “Terminal” ) and follow the steps below to first install the support software for Quire, and then Quire itself. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
 
-1. **Apple’s Xcode** is a set of developer tools for your mac. More information can be found [here](https://en.wikipedia.org/wiki/Xcode). Install Apple's Xcode by copying and pasting the following command and pressing enter. If Xcode is not already installed, an additional alert notification will pop up. Click “Install” and follow the prompts.
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/documentation/install-uninstall.md:16]: I should double check with Matthew but this is correct, it's no longer needed. We were only having them install xcode to give them access to the `git` command in Terminal so that they could run `git clone https://github.com/thegetty/quire-cli.git`.

File: content/documentation/install-uninstall.md [hunk 4]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
     ```text
-    git clone https://github.com/gettypubs/quire-cli.git
+    cd Downloads/prince
+    sudo ./install.sh
     ```
 
-    You may be asked to enter your GitHub username and password. When complete, install the CLI by copying and pasting the following commands into your Terminal and pressing enter. The first resets user permissions for your local directory, the second installs Quire.
+    You will be prompted to enter your computer password. Press enter. You should receive a message that PrinceXML will be installed in the `/usr/local` directory. Press enter again. If successful you will see a message in the Terminal saying "installation complete." Or, if you get a "no such file or directory" message after the first line, the file you dowloaded is either not in your Downloads folder, or is not named "prince". Correct as necessary and try again.
+
+    When complete, type `cd` into the Terminal to return to your home directory.
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/documentation/install-uninstall.md:47]: Good point. We have it already defined in the glossary, so I can add a glossary pop-up here

File: content/documentation/install-uninstall.md [hunk 5]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
 
     ```text
-    sudo chown -R $USER /usr/local
+    cd
     ```
 
+5. The **Quire CLI** ({{< q-def "command-line interface" >}}) is the core of Quire. It is used for creating, previewing, and outputting projects. You can learn more about the Quire CLI in the [*Quire CLI Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into your Terminal to install.
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/documentation/install-uninstall.md:53]: Good point. I don't want to totally obscure the fact of the CLI but we should definitely lean toward just calling Quire Quire. Let me rework this section and see if I can get anywhere better with it.

File: content/documentation/install-uninstall.md [hunk 6]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+
     ```text
-    cd quire-cli
-    npm install -g
+    npm install --global @thegetty/quire-cli
     ```
 
-When complete, type `quire --version` to confirm proper installation. It should return a version number, otherwise, if it says `command not found`, the Quire CLI has not been properly installed. Refer to the [*Troubleshooting*](#troubleshooting) section below.
+The CLI installation process may take several minutes, during which time there will be considerable messaging output in Terminal. The only messages of any concern are those labeled as Error or ERR. These likely indicate a failed installation, as would seeing "command not found" after entering `quire --help` in step 3. Refer to the [*Troubleshooting*](#troubleshooting) section below.
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/documentation/install-uninstall.md:59]: Oh yes, referring people to the Forum is the way to go. I'll make some changes accordingly.

File: content/documentation/install-uninstall.md [hunk 7]
```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
 
-    ```text
-    xcode-select --install
-    ```
+If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 2–4 in the Full Install guidelines below.
 
-2. **Node.js** enables you to run javascript on your computer. Visit the Node.js site, and download and install the current LTS (long-term support) version: [https://nodejs.org](https://nodejs.org).
+1. Install the **LTS** version of Node.js: [https://nodejs.org](https://nodejs.org)
+2. In Terminal, install the Quire CLI with: `npm install --global @thegetty/quire-cli`
+3. Confirm installation: `quire --help`
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/documentation/install-uninstall.md:22]: I changed this to `--help` because I actually thought that would be a good way to teach people that this command brings up a list of all the quire commands. But what do you think? Here's what it looks like now, and we could definitely work with Hannah and Matthew to add to or expand on this:

```
galbers@GT29043 quire-docs % quire --help
Usage: quire [options] [command]

Options:
  -V, --version                output the version number
  -v, --verbose                log verbose output
  -f, --file                   Add Filename and optional new filepath
  -e, --env                    Add environment variable
  -h, --help                   output usage information

Commands:
  new <projectName>            Create a new Quire project in the current directory.
  preview [options]            Run the preview server in the current directory
  install                      Install this project's theme dependencies
  site|build [options] [env]   Run the build command in the current directory
  pdf [options] [file] [env]   Generate a PDF version of the current project
  epub [options] [file] [env]  Generate an EPUB version of the current project
  mobi [options] [file] [env]  Generate an MOBI version of the current project
  template [options] [type]    Download templates to customize your file output (Currently only EPUB supported)
  debug                        Development use only - log info about current project
```

File: content/documentation/install-uninstall.md (hunk 2 above)
Comment (geealbers) [reply content/documentation/install-uninstall.md:9]: Agreed. This was just a starting point for sure. And in fact the bit about access to staff maybe isn't necessary here at all. Happy for any edits or suggestions. Maybe we can go over it on our Monday call.

File: content/documentation/install-uninstall.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Erin-Cecele) [reply content/documentation/install-uninstall.md:22]: I agree! I would suggest editing to say something along the lines of:

Step. 3 - Run the command `quire --help.` This will give you a list of commands that will help you get started using and navigating Quire. More can be found in the Quire Commands section of the docs. And if you get "command not found" it means it was not installed correctly. See Troubleshooting section, etc. 

When I used Quire for the first time and ran the version command as the final install step, I was so thrilled to know I did it correctly and didn't get the error message. It gave me a sense of accomplishment and good momentum moving forward. Then they'd also know that, oops, if they did get an error message they can check Troubleshooting.

Maybe we include a note in Troubleshooting section directing people to the forum.

File: content/documentation/install-uninstall.md (hunk 2 above)
Comment (Erin-Cecele) [reply content/documentation/install-uninstall.md:9]: Sounds good! For now, maybe we could just remove the Getty staff part and leave it at "access to support" because that could include the forum. We can explain in more depth in the actual sign-up form. (I've made notes on how we might edit the form which we can go over on Monday!)

**Comment by geealbers** [comment]
@Erin-Cecele this is ready for a second, final review from you. I've left the rather bonkers Linux install as is for now because I really have no idea about it. Maybe just better to remove it altogether for now. I'm not sure

File: content/documentation/install-uninstall.md [hunk 8]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+### Quick Install
+
+If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 3–5 in the Full Install guidelines below.
+
+1. In Terminal, install Apple’s Xcode with: `xcode-select --install`
+2. Download and install the **LTS** version of Node.js: [https://nodejs.org](https://nodejs.org)
+3. In Terminal, install Quire with: `npm install --global @thegetty/quire-cli`
+4. Confirm installation: `quire --help`
+
+The Quire installation process may take several minutes, during which time there will be considerable messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation, as would seeing "command not found" after entering `quire --help` in step 4. Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:25]: Can we have this match lines 66 and 68? I'm fine with it if this is your preference, I just like the consistent messaging and explanation of the commands.

File: content/documentation/install-uninstall.md [hunk 9]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+2. **Node.js** enables you to run javascript on your computer. Visit the Node.js site, and download and install the older LTS (long-term support) version, which is sufficient and more stable than the higher “Current” version that is also available for download: [https://nodejs.org](https://nodejs.org).
 
-3. **Pandoc** serves two purposes in Quire. You can use it to convert Word documents to Markdown, and you can also use it to create e-book files of your Quire project. Visit the Pandoc repository on GitHub, and download the macOS .pkg file of the latest release: [https://github.com/jgm/pandoc/releases/](https://github.com/jgm/pandoc/releases/). Install by double clicking the icon and following the prompts.
+3. **Pandoc** serves two purposes in Quire: You can use it to [convert Word documents to Markdown](/fundamentals/#microsoft-word-to-markdown-conversion), and it enables you to create EPUB e-book files of your Quire project with the `quire epub` command. Download the macOS installer, double-click it and follow the prompst to install: https://pandoc.org/installing.html
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:39]: prompts is misspelled.

File: content/documentation/install-uninstall.md [hunk 10]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
 
-4. **PrinceXML** enables you to create a PDF version of your Quire project. Visit the PrinceXML site, download the Mac OS version, and uncompress the folder: [http://www.princexml.com/download/](http://www.princexml.com/download/).
+4. **Kindle Previewer**, along with Pandoc, enables you to create MOBI e-book files of your Quire project with the `quire mobi` command. Visit Amazon’s Kindle Previewer page and downoad the Mac version: https://kdp.amazon.com/en_US/help/topic/G202131170. Install by double clicking the icon and following the prompts.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:41]: download is misspelled

File: content/documentation/install-uninstall.md [hunk 11]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+6. **Quire** is operated through a {{< q-def "command-line interface" >}} (CLI) that enables you to create, preview and output publications using Terminal with commands like `quire new`, `quire preview` and `quire site`. You can learn more in the [*Quire Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into Terminal to install Quire:
 
     ```text
-    git clone https://github.com/gettypubs/quire-cli.git
+    npm install --global @thegetty/quire-cli
     ```
 
-    You may be asked to enter your GitHub username and password. When complete, install the CLI by copying and pasting the following commands into your Terminal and pressing enter. The first resets user permissions for your local directory, the second installs Quire.
+    The Quire installation process may take several minutes, during which time there will be considerable messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:66]: Is this still true? I didn't get as much messaging as the first time and it went pretty quick.

File: content/documentation/install-uninstall.md [hunk 12]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+    To verify installation, enter the command below. This will give you a list of [commands](/documentation/quire-cli/) that will help you get started using and navigating Quire. And if you get "command not found" it means it was not installed correctly.
 
     ```text
-    cd quire-cli
-    npm install -g
+    quire --help
     ```
 
-When complete, type `quire --version` to confirm proper installation. It should return a version number, otherwise, if it says `command not found`, the Quire CLI has not been properly installed. Refer to the [*Troubleshooting*](#troubleshooting) section below.
+    Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:74]: Once we have our Get Started page figured out should we maybe link to it here as a next step now that they've installed? I can just make a note to add this in later.

Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:74]: We should also add the link to the Troubleshooting section of the Forum once that is up and running. I can just make a note to add this in later.

File: content/documentation/install-uninstall.md [hunk 13]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+1. Download and install Git for Windows: https://gitforwindows.org/
+2. Download and install the **LTS** version of Node.js: [https://nodejs.org](https://nodejs.org)
+3. In Windows PowerShell (Admin), install Windows Build Tools with: `npm install --g --production windows-build-tools`
+4. In Windows PowerShell (Admin), install Quire with: `npm install --global @thegetty/quire-cli`
+5. Confirm installation: `quire --help`
 
-5. Open **PowerShell** (that should be installed by default in your Windows computer) and run it as administrator. To do so, right click on the icon and select "run as administrator" from the context menu.
+The Quire installation process may take several minutes, during which time there will be considerable messaging output in PowerShell. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation, as would seeing "command not found" after entering `quire --help` in step 5. Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:90]: and would say "Command not found" ?

File: content/documentation/install-uninstall.md [hunk 14]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
...
+    To verify installation, enter the command below. This will give you a list of [commands](/documentation/quire-cli/) that will help you get started using and navigating Quire. And if you get "command not found" it means it was not installed correctly.
 
-    ```tx
-    cd ~
+    ```text
+    quire --help
     ```
 
+    Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
+
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:127]: Once we have our Get Started page figured out should we maybe link to it here as a next step now that they've installed? Same with Troubleshooting section on Forum.

File: content/documentation/install-uninstall.md [hunk 15]
```diff
@@ -248,133 +257,41 @@ If version number is returned, quire-cli was installed correctly. You can now le
...
-
-1. In the `themes` directory of your project, delete the current theme directory. It’s `quire-starter-theme` by default.
-2. Copy in your new theme directory ensuring that it has the same name as the original.
-3. Open your command-line shell and navigate to the project directory using the `cd` (change directory) command. For example, if your project directory was called `my-project`, and it was in your main user directory, you’d enter `cd my-project`.
-3. Still in the command-line shell, type `quire install` and press enter to install the theme dependencies for your project. (This is done automatically when running `quire new`, but needs to be done manually when adding a new or replacement theme.)
+## Update Quire
 
 {{< q-class "box warning" >}}
-- Be sure to save any [customizations you’ve made](/documentation/styles-customization/) inside your theme. (Typically style changes to the `variables.scss` file.) You’ll have to copy these over into the new theme manually once it is installed.
+- Updating Quire will not effect your existing projects. It swill only take effect on any new projects you start after updating.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:263]: "It will"

File: content/documentation/install-uninstall.md [hunk 16]
```diff
@@ -248,133 +257,41 @@ If version number is returned, quire-cli was installed correctly. You can now le
...
-
-8. This page allows you to select the scopes or permissions you'd like to grant this token. In this instance, click **repo** at the very top.
-
-9. Scroll down to the bottom of the page and click **generate token**.
+```text
+npm install --global @thegetty/quire-cli
+```
 
-10. Copy the token to your clipboard before closing this window. For security reasons, after you navigate off the page, you will not be able to see the token again.(Treat your tokens like passwords, and keep them secret.)
+Note that while any *new* projects you start at this point will use the latest version of Quire just installed, older, previously started projects will remain untouched. This to avoid any unforeseen and unintended changes to publications that are completed or in progress as the version of Quire you use can effect layouts of a page, availablility of features, etcetera.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:272]: "This is to avoid"

File: content/documentation/install-uninstall.md [hunk 17]
```diff
@@ -248,133 +257,41 @@ If version number is returned, quire-cli was installed correctly. You can now le
...
 
-**Resetting user permission for local directory**
+run commands from the new version of Quire on old Quire projects.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:274]: I'm not sure what this line is? It's it a section header?

File: content/documentation/install-uninstall.md [hunk 18]
```diff
@@ -248,133 +257,41 @@ If version number is returned, quire-cli was installed correctly. You can now le
...
 
-When you have finished these steps, type the following command to confirm proper installation:
+```text
+npm uninstall --global @thegetty/quire-cli
+```
 
-  ```text
-  quire --version
-  ```
+This will only uninstall Quire itself. External software you installed as part of Quire (inlcuding Node.js, Pandoc, Kindle Previewer, and PrinceXML) will need to be uninstalled individually according to the directions they provide.
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:297]: It might be nice to link to those instructions if that is doable, I was trying to uninstall on my personal laptop and got a little lost. Maybe just something to consider as a nice addition in the future and not critical now.

File: content/documentation/install-uninstall.md [hunk 19]
```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 ---
 
 {{< q-class "box warning" >}}
-- Quire is in a closed beta. Installation will not work unless you have [requested and been given access](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
+- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open source software. For a free license to use Quire for your publication projects (and for additional access to support) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
```
Comment (Erin-Cecele) [inline content/documentation/install-uninstall.md:9]: I'll provide you with the new link to the Beta Form once I have it.

**Review by Erin-Cecele — COMMENTED** [review]
I think this looks great. I mostly just caught some typos and had a few small questions. I'm ok with removing the Linux section. If I remember correctly we had a handful of Linux users but it's pretty rare.

File: content/documentation/install-uninstall.md (hunk 11 above)
Comment (geealbers) [reply content/documentation/install-uninstall.md:66]: Changed this to "a minute or two" and took out "considerable"

File: content/documentation/install-uninstall.md (hunk 13 above)
Comment (geealbers) [reply content/documentation/install-uninstall.md:90]: I don't think I understand your query here. Is it just that I should capitalize Command?

File: content/documentation/install-uninstall.md (hunk 8 above)
Comment (geealbers) [reply content/documentation/install-uninstall.md:25]: Match in what way? I saw 66 and 68 as a fuller version of the process. I used here verbatim on line 66, but I can't change 68 because they haven't yet run `quire --help`

File: content/documentation/install-uninstall.md (hunk 12 above)
Comment (geealbers) [reply content/documentation/install-uninstall.md:74]: Yes, would be good to give them a dirct path to whatever we think they'll want to do next

File: content/documentation/install-uninstall.md (hunk 17 above)
Comment (geealbers) [reply content/documentation/install-uninstall.md:274]: just a mistake

File: content/documentation/install-uninstall.md (hunk 13 above)
Comment (Erin-Cecele) [reply content/documentation/install-uninstall.md:90]: Ah, I reread it a few times... the phrasing "as would seeing" confused me but it does make sense. My apologies!

Comment (geealbers) [reply content/documentation/install-uninstall.md:90]: Oh! How about:

> If you see these errors, or if you see "command not found" after entering `quire --help` in step 5., search our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.

File: content/documentation/install-uninstall.md (hunk 8 above)
Comment (Erin-Cecele) [reply content/documentation/install-uninstall.md:25]: I think Step. 4 is a little confusing because we aren't telling them what to expect by running that command other than you might see "command not found." 

I can see how it might be too wordy but I feel like we should incorporate "You should see  a list of [commands](/documentation/quire-cli/) that will help you get started using and navigating Quire." I can imagine some people might get confused when they run that command and a bunch of text pops up in their terminal.

File: content/documentation/install-uninstall.md (hunk 13 above)
Comment (Erin-Cecele) [reply content/documentation/install-uninstall.md:90]: Yes, I like "if you see 'command not found'..."
